playbooks/openstack-osh_hostconfig.yml

```diff
@@ -14,12 +14,17 @@
         state: present
       loop: "{{ deploy_on_openstack_osh_repos_per_imagename[deploy_on_openstack_oshnode_image] }}"
 
+    - name: Refresh all repositories
```

Comment:
I think we should do this instead:

- name: Add known vendors
  blockinfile:
    path: /etc/zypp/vendors.d/opensuse
    block: |
      [main]
      vendors = "suse,opensuse,obs://build.opensuse.org/Cloud:OpenStack,obs://build.opensuse.org/devel:languages:python"
    create: yes